Alignment/CommonAlignmentProducer/plugins/AlignmentProducer.cc

```diff
@@ -641,7 +641,7 @@ void AlignmentProducer::simpleMisalignment_(const Alignables &alivec, const std:
 
 
 //__________________________________________________________________________________________________
-void AlignmentProducer::createGeometries_( const edm::EventSetup& iSetup )
+void AlignmentProducer::createGeometries_( const edm::EventSetup& iSetup, const TrackerTopology* tTopo )
```

Comment:
@ianna are you fine with this? this is the only change that is touching alca packages